main.cpp

```diff
@@ -52,106 +52,116 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: student
+//  action 1: the student study
+student.study();
+//  action 2: the student rest
+student.rest();
+//  action 3: the student sleep
+student.sleep();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: employee
+//  action 1: the employee works
+employee.work();
+//  action 2: the employee eats
+employee.eat();
+//  action 3: the employee goes back home
+employee.goesBackHome();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: computer
+//  action 1: the computer calculates
+computer.calculate();
+//  action 2: the computer memorizes
+computer.memorize();
+//  action 3: the computer consumes energy
+computer.consumeEnergy();
+
+//  4) 
+//  Noun: car
+//  action 1: the car running
+car.running();
+//  action 2: the car consumes fuel 
+car.consumeFuel();
+//  action 3: the car close the door
+car.closeDoor();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bird
+//  action 1: the bird flies
+bird.fly();
+//  action 2: the bird eats
+bird.eat();
+//  action 3: the bird protects itself
+bird.protectItSelf();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

Reply:
Thanks for your revision, Chuck..... but, I think that I don't understand something in whole procces... all the names  that I wrote begining in lowercase, and, if there a second word, in uppercase: bird, eat, protectItSelf, following that I understand about camelCase.